Makefile

```diff
@@ -36,21 +40,31 @@ build-cross:
 	$(call go-build,windows,386,${BUILDTAGS})
 
 .PHONY: all
-all: build-amd64 build-386
+all: build-amd64 build-386 build-amd64-cni build-386-cni
 
 .PHONY: build
-build: build-amd64 build-386
+build: build-amd64 build-386 build-amd64-cni build-386-cni
 
 .PHONY: build-amd64
 build-amd64:
 	GOARCH=amd64 $(GO_BUILD) -tags $(BUILDTAGS) ./...
 
+.PHONY: build-amd64
+build-amd64-cni:
+	GOARCH=amd64 $(GO_BUILD) -tags $(BUILDTAGS),cni ./...
+
 .PHONY: build-386
 build-386:
 ifneq ($(shell uname -s), Darwin)
 	GOARCH=386 $(GO_BUILD) -tags $(BUILDTAGS) ./...
 endif
 
+.PHONY: build-386-cni
+build-386-cni:
+ifneq ($(shell uname -s), Darwin)
+	GOARCH=386 $(GO_BUILD) -tags $(BUILDTAGS),cni ./...
+endif
```

Comment:
lets just remove this target testing amd64 should be enough for this build tag, if anyone every tries to add arch specific code there I will go crazy.